phpunit.xml.dist

```diff
@@ -1,16 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd" bootstrap="vendor/autoload.php" colors="true">
-  <coverage processUncoveredFiles="true">
-    <include>
-      <directory suffix=".php">./src</directory>
-    </include>
-    <exclude>
-      <directory suffix=".php">./src/MezzioInstaller/Resources</directory>
-    </exclude>
-  </coverage>
-  <testsuites>
-    <testsuite name="App\\Tests">
-      <directory>./test</directory>
-    </testsuite>
-  </testsuites>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+         bootstrap="vendor/autoload.php"
+         cacheResultFile=".phpunit.cache/test-results"
+         executionOrder="depends,defects"
+         beStrictAboutCoversAnnotation="true"
+         beStrictAboutOutputDuringTests="true"
+         beStrictAboutTodoAnnotatedTests="true"
+         failOnRisky="true"
+         failOnWarning="true"
+         verbose="true"
+        colors="true">
+    <testsuites>
+        <testsuite name="default">
+            <directory suffix="Test.php">test</directory>
```

Comment:
The `suffix` attribute can be omitted because it is [redundant with the value `Test.php`](https://github.com/sebastianbergmann/phpunit/blob/8e16c225d57c3d6808014df6b1dd7598d0a5bbbe/schema/9.2.xsd#L168).